amundsen_application/static/js/config/index.spec.ts

```diff
@@ -132,6 +132,7 @@ describe('getNavLinks', () => {
 
   describe('when announcements is active', () => {
     it('returns all the navLinks', () => {
+      AppConfig.announcements.enabled = true;
```

Comment:
I notice here you are just following other examples this test file is using by directly setting the value needed for the test, however this habit can cause side effects in other tests because the modifications will apply across all unit tests in this file.

It's been on my todo list to update these tests to show how to properly mock the object, so I will submit a PR this week for that and it can be used as an example moving forward.

Reply:
oh cool, are all the JS tests executed in parallel or in sequential order?